### ISSUE TITLE
pip install from pypi fails due to inconsistent version numbering

### ISSUE THREAD

**Issue by ebellm** [blanton144/kcorrect#4]
`pip install kcorrect` currently fails with the following error:

```
Requested kcorrect from https://files.pythonhosted.org/packages/6d/33/88d6cf0e01aff4d97b76bcb8ef4ab659413cdbd7b0c016cc73920ac8e52f/kcorrect-5.0.0b3.tar.gz has inconsistent version: filename has '5.0.0b3', but metadata has '5.0.0b0'
ERROR: Could not find a version that satisfies the requirement kcorrect (from versions: 5.0.0b3)
ERROR: No matching distribution found for kcorrect
```

**Comment by blanton144** [comment]
Hi--yes this seems to happen on some systems. I am not sure why PYPI
allowed me to upload the file but doesn't allow downloading. I have not had
spare time since August to fix this, but I will on the next tag. You should
be able to clone the repo and "pip install ." ... Sorry for the
inconvenience!

**Comment by blanton144** [comment]
Dear Eric,

I've finally (with some help from the LINCC people at UW) fixed this (I
think) and also cleaned up some other issues.

I'd appreciate it if you try installing and using the pre-release
kcorrect==5.1.0rc1 to see if it is ok!

Thanks,
Mike

On Mon, Jan 30, 2023 at 7:55 AM Michael Blanton ***@***.***>
wrote:

> Hi--yes this seems to happen on some systems. I am not sure why PYPI
> allowed me to upload the file but doesn't allow downloading. I have not had
> spare time since August to fix this, but I will on the next tag. You should
> be able to clone the repo and "pip install ." ... Sorry for the
> inconvenience!
>
> On Sun, Jan 29, 2023, 8:55 PM Eric Bellm ***@***.***> wrote:
>
>> pip install kcorrect currently fails with the following error:
>>
>> Requested kcorrect from https://files.pythonhosted.org/packages/6d/33/88d6cf0e01aff4d97b76bcb8ef4ab659413cdbd7b0c016cc73920ac8e52f/kcorrect-5.0.0b3.tar.gz <https://urldefense.proofpoint.com/v2/url?u=https-3A__files.pythonhosted.org_packages_6d_33_88d6cf0e01aff4d97b76bcb8ef4ab659413cdbd7b0c016cc73920ac8e52f_kcorrect-2D5.0.0b3.tar.gz&d=DwQCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=2nqg0fcNitaQTtJ4c9MSjg&m=mfMg7mguniwPeZi68P3pnxlk7YAJqG6q2w5dprkWGjTSByDT7Zo96AGPMx3TSuHx&s=S2O7IBQUs_uGHloJu8asSSgt3HQHDYwcMF0Y-Iph66Y&e=> has inconsistent version: filename has '5.0.0b3', but metadata has '5.0.0b0'
>> ERROR: Could not find a version that satisfies the requirement kcorrect (from versions: 5.0.0b3)
>> ERROR: No matching distribution found for kcorrect
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_blanton144_kcorrect_issues_4&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=2nqg0fcNitaQTtJ4c9MSjg&m=mfMg7mguniwPeZi68P3pnxlk7YAJqG6q2w5dprkWGjTSByDT7Zo96AGPMx3TSuHx&s=YYgV2jrlX5YkpCDM_kST-_gph5oPOx8_DChQ_0BNV_o&e=>,
>> or unsubscribe
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AAODDFSYKFPXE3YQJO4VXETWU4NQNANCNFSM6AAAAAAUKS2XMA&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=2nqg0fcNitaQTtJ4c9MSjg&m=mfMg7mguniwPeZi68P3pnxlk7YAJqG6q2w5dprkWGjTSByDT7Zo96AGPMx3TSuHx&s=BUaOmPFx5nc44zG60FjUkJjI16f1CdZpzKUSMnBcyAg&e=>
>> .
>> You are receiving this because you are subscribed to this thread.Message
>> ID: ***@***.***>
>>
>

-- 
Michael Blanton, Professor
Department of Physics
New York University

**Comment by JonLoveday** [comment]
Hi Mike

It looks like in kcorrect==5.1.0rc1 you have updated the format of the response files from .par to .dat.  I have some .par files for other systems (VST, VISTA).  Do you have a script for converting from .par to .dat?

Thanks, Jon

**Comment by blanton144** [comment]
Dear Jon,

Yes ... That's a pre-release so not ready for prime-time quite yet.

I do have a script for converting .par to .dat (which depends on pydl of
course ...). I can put it in a more obvious place in the repo, but if you
want to dig it is in the data/responses directory ...

Thanks,
Mike

On Tue, Sep 26, 2023 at 12:12 PM JonLoveday ***@***.***>
wrote:

> Hi Mike
>
> It looks like in kcorrect==5.1.0rc1 you have updated the format of the
> response files from .par to .dat. I have some .par files for other systems
> (VST, VISTA). Do you have a script for converting from .par to .dat?
>
> Thanks, Jon
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_blanton144_kcorrect_issues_4-23issuecomment-2D1735859183&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=2nqg0fcNitaQTtJ4c9MSjg&m=clRE-NG2d9TREBhE-ippZMSiEs5oxiEgpCdkb1kpCmk48uJohb5mrpQ0x36kV7Cg&s=X2ReBUIdILEM8XzvedpBLw4bse6ailaHU2MPrbHZWPA&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AAODDFWQG34RYDRAEBFF4WDX4L5HDANCNFSM6AAAAAAUKS2XMA&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=2nqg0fcNitaQTtJ4c9MSjg&m=clRE-NG2d9TREBhE-ippZMSiEs5oxiEgpCdkb1kpCmk48uJohb5mrpQ0x36kV7Cg&s=AA4t0v1fyH7O1sAKuNxZucGSEBNAP2hwD0f3y34itSQ&e=>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Michael Blanton, Professor
Department of Physics
New York University

**Comment by JonLoveday** [comment]
Thanks Mike

Note that 'pip install kcorrect' gives 5.1.0rc1, as my student just found out.

Cheers, Jon

**Comment by blanton144** [comment]
This is fixed as of 5.1.0
